packages/synapse-react-client/src/components/SynapseForm/SynapseForm.tsx

```diff
@@ -937,6 +938,11 @@ export default class SynapseForm extends React.Component<
                     disabled={
                       this.state.currentStep.excluded || this.state.isSubmitted
                     }
+                    widgets={{
+                      CheckboxWidget: Widgets.CheckboxWidget,
+                      CheckboxesWidget: Widgets.CheckboxesWidget,
+                      RadioWidget: Widgets.RadioWidget,
+                    }}
```

Comment:
By default, RJSF uses Bootstrap classes, but the bootstrap checkbox/radio classes don't play nicely with SynapseForm styles without our hacky custom checkbox styles.

It would be great to just use the RJSF MUI theme, but some aspects of that also don't look good with the SynapseForms style.

What will work nicely for now is to pull in and use the specific checkbox/radio widgets from MUI, but leave everything else the same.

https://github.com/user-attachments/assets/84a2a66b-0f6e-41d0-922d-ebfba5d34a2f

Reply:
https://github.com/user-attachments/assets/adf14f33-cff0-4960-a5cd-d792d32e85cf